sdk/docs/iot/README.md

```diff
@@ -71,6 +71,30 @@ Feature | Azure SDK for Embedded C | Description
  Retry Policies | &radic;* | The Azure SDK for Embedded C provides guidelines for retries, but actual retries should be handled by the application.
  [IoT Plug and Play](https://docs.microsoft.com/azure/iot-pnp/overview-iot-plug-and-play) | &radic; | IoT Plug and Play enables solution builders to integrate smart devices with their solutions without any manual configuration.
 
+<br>
+
+## Size Chart
+
+The following chart shows the RAM and ROM usage for the PIC24-IOT dev kit from Microchip found in https://github.com/Azure-Samples/Microchip-PIC-IoT-Wx compiled using Microchip’s XC16 version 1.60 compiler with the following macros: AZ_NO_LOGGING, AZ_NO_CONDITION_CHECKING and NDEBUG.
+
+This PIC24-IOT kit has 2 MCUs (microcontrollers):
+ - Host MCU, a 16-bit **PIC24FJ128GA705** with 128 KB Flash Memory & 16 KB SRAM: this MCU runs the Embedded C SDK, the sample code and the MQTT client provided by Microchip.
+ - Wi-fi MCU, an **ATWINC**1510 Wi-Fi Module: this MCU runs the TLS and TCP stack. 
+
+The Pic24 sample includes both Hub and DPS services. The table below shows RAM/ ROM sizes considering:
+
+-  Embedded C SDK libraries only – which represent the baseline size 
+-  Total size – which includes the Embedded C SDK and Microchip MQTT client (this Dev Kit has a separate Wi-Fi module which runs the TLS and TCP/IP stacks).
+
+|  | Embedded C SDK size | | Total Size | |
+|---------|----------|---------|---------|---------
+|**Sample** | **Program/ROM** | **Data/RAM** | **Program/ROM** | **Data/RAM** | 
+| PIC24 (Hub + DPS + IoT Plug and Play) | 26.15KB | 0 | 103.61KB | 10.57KB
+| PIC24 Telemetry only | 2.58KB | 0 | 74.16KB | 8.26KB
```

Comment:
Just seeing absolute numbers, the question comes to mind: Are these numbers good? What's the goal or comparison to the alternative/previous, which would highlight the value of the new Embedded SDK?

Reply:
Comparison shows the memory usage when you have only telemetry x all features. It highlights how small of a footprint can be used in a device with the Embedded C SDK.